app/routers/api/identity.py

```
@@ -0,0 +1,98 @@
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="Invalid password",
+        )
+
+    return identity_service.export_keys_pem(db)
+
+
+@router.post("/import", status_code=status.HTTP_200_OK)
+def import_identity(
+    request: IdentityImportRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)